todos-app/src/App/TodoList/TodoListItem/TodoListItem.tsx

```diff
@@ -1,49 +1,31 @@
-import React, { useState, createRef } from "react";
-import { ITodo } from "../../../DataStructure";
+import { Todo } from "../../models/Todos";
 import "../../../App.css";
 
-type TodoListItemsProps = {
-  todo: ITodo;
+type Props = {
+  todo: Todo;
   onToggle(id: string): void;
   onRemove(id: string): void;
 };
 
-interface State {
-  onEdit: boolean;
-}
-
-const TodoListItem: React.FC<TodoListItemsProps> = ({
-  todo,
-  onToggle,
-  onRemove,
-}) => {
-  const init: State = { onEdit: false };
-  const [state, setState] = useState(init);
-
-  const SwitchStyle = (t: ITodo, onEdit: boolean): string => {
-    switch (true) {
-      case onEdit && t.completed:
-        return "completed editing";
-      case onEdit && !t.completed:
-        return "editing";
-      case !onEdit && t.completed:
-        return "completed";
-      case !onEdit && !t.completed:
-        return "";
-
-      default:
-        return "";
+const TodoListItem = ({ todo, onToggle, onRemove }: Props) => {
+  const switchStyle = (t: Todo): string => {
+    if (t.completed) {
+      return "completed";
+    } else {
+      return "";
     }
   };
 
   return (
-    <li key={todo.id} className={SwitchStyle(todo, state.onEdit)}>
+    <li key={todo.id} className={switchStyle(todo)}>
```

Comment:
you can also try one liner `className={todo.completed && "completed"}` or `className={todo.complete ? "completed" : ""}`